libkbfs/journal_server.go

```diff
@@ -303,12 +291,15 @@ func (j *JournalServer) Disable(ctx context.Context, tlfID TlfID) (
 			"are any dirty blocks outstanding", tlfID)
 	}
 
-	tlfJournal.shutdown()
-
-	j.log.CDebugf(ctx, "Disabled journal for %s", tlfID)
+	wasEnabled, err = tlfJournal.disable()
```

Comment:
I was gonna comment that perhaps you should disable and remove from `tlfJournals`, but then I thought about it more and realized that if we did that, and something held on to the old `tlfJournal` for too long, then even if it's re-enabled, using the old `tlfJournal` would hit the server. I guess you had this in mind? Might be worth a comment.